examples/sample-app/container-setup.md

```diff
@@ -76,8 +71,7 @@ step #3.
 
 ## Deploy the private docker registry
 
-    $ chmod +r ./openshift.local.certificates/openshift-client/key.key
-    $ CERT_DIR=openshift.local.certificates/openshift-client ./install-registry.sh
+    $ openshift ex registry --create --credentials="${KUBECONFIG}"
```

Comment:
There's not specific `export` in this doc for `KUBECONFIG`. I do see it in the referenced doc below (app build/deploy/update step 4) - do you think we need to explicitly add back setting `KUBECONFIG` in this doc?